modules/nrfxlib/nrf_802154/sl/platform/nrf_802154_platform_sl_lptimer_zephyr.c

```diff
@@ -98,40 +114,150 @@ static void timer_start_at(struct timer_desc *timer, uint64_t target_time)
 	nrf_802154_sl_mcu_critical_exit(state);
 }
 
+static inline uint32_t rtc_counter_diff(uint32_t a, uint32_t b)
+{
+	return (a - b) & RTC_COUNTER_MAX;
```

Comment:
`RTC_COUNTER_MAX` is provided only under `#if defined(CONFIG_SOC_NRF5340_CPUNET)`, doesn't build